CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 Changes since v1.3.0-rc.2
 
+- Remove unneeded 32-bit specific lock types.
```

Comment:
This should say how it affects the user:
```
- Remove unneeded 32-bit specific lock types that were changed in 1.3.0-rc.2, because they interfered with 32-bit builds.
```

Reply:
thanks for the comment. Updated in the code.